torch/utils/hooks.py

```diff
@@ -58,3 +61,122 @@ def warn_if_has_hooks(tensor):
                               "serialized.  If this is expected, you can "
                               "decorate the function with @torch.utils.hooks.unserializable_hook "
                               "to suppress this warning".format(repr(hook)))
+
+class BackwardHook(object):
+    """
+    A wrapper class to implement nn.Module backward hooks.
+    It handles:
+      - Ignoring non-Tensor inputs and replacing the, by None before calling the user hook
```

Comment:
Missing something after "replace the"

Reply:
Ho qwerty keyboard where `m` is `,` updated.